src/compiler/transformers/utilities.ts

```diff
@@ -297,41 +297,29 @@ namespace ts {
     }
 
     /**
-     * Gets all property declarations with initializers on either the static or instance side of a class.
+     * Gets all the static or all the instance property declarations of a class
      *
      * @param node The class node.
      * @param isStatic A value indicating whether to get properties from the static or instance side of the class.
      */
-    export function getInitializedProperties(node: ClassExpression | ClassDeclaration, isStatic: boolean): readonly PropertyDeclaration[] {
-        return filter(node.members, isStatic ? isStaticInitializedProperty : isInstanceInitializedProperty);
+    export function getProperties(node: ClassExpression | ClassDeclaration, requireInitializer: boolean, isStatic: boolean): readonly PropertyDeclaration[] {
+        return filter(node.members, m => isInitializedOrStaticProperty(m, requireInitializer, isStatic)) as PropertyDeclaration[];
     }
 
     /**
-     * Gets a value indicating whether a class element is a static property declaration with an initializer.
+     * Is a class element is either a static or an instance property declaration with an initializer?

```

Comment:
```suggestion
     * Is a class element either a static or an instance property declaration with an initializer?
```

Reply:
Evidence for [movement in minimalism](https://www.oxfordhandbooks.com/view/10.1093/oxfordhb/9780199549368.001.0001/oxfordhb-9780199549368-e-009)????